src/screens/GroupInfo/elements/Link.js

```diff
@@ -35,7 +36,7 @@ export default Link;
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on replacing the hard-coded color value with a constant from the theme file. This will improve maintainability and consistency across the application.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->